module_cheatsheets/scRNA-seq_cheatsheet.md

```diff
@@ -0,0 +1,81 @@
+
+<div style="page-break-after: always;"></div>
+
+### `scran`, `scater`, `SingleCellExperiment`
+Read the `scran` package documentation [**here**](https://www.rdocumentation.org/packages/scran/versions/1.10.2), and a vignette on its usage [**here**](https://bioconductor.org/packages/devel/bioc/vignettes/scran/inst/doc/scran.html).
+
+Read the `scater` package documentation [**here**](https://www.rdocumentation.org/packages/scater/versions/1.10.1), and a vignette on its usage [**here**](https://github.com/davismcc/scater/blob/master/vignettes/vignette-intro.Rmd).
+
+Read the `SingleCellExperiment` package documentation [**here**](https://osca.bioconductor.org/), and a vignette on its usage [**here**](https://bioconductor.org/packages/devel/bioc/vignettes/SingleCellExperiment/inst/doc/intro.html).
+
+<br>
+
+| Library/Package      | Piece of Code              | What it's called                           | What it does                                                                                                           |
+|----------------------|----------------------------|--------------------------------------------|--------------------------------------------------------------|
+| `scran`                | [`quickCluster()`](https://www.rdocumentation.org/packages/scran/versions/1.0.3/topics/Quick%20clustering)                | Quick Clustering     | Groups similar cells into clusters and normalizes these cells separately. The size factors are rescaled for comparison across clusters.                               |
+| `scran`                | [`computeSumFactors()`](https://bioconductor.org/packages/devel/bioc/vignettes/scran/inst/doc/scran.html)                 | Compute Sum Factors        | Returns a numeric vector of computed sum factors for each cell cluster in the object. The cluster-based size factors are deconvolved into cell-based size factors for the normalization of each cell's gene expression profile. This is done to avoid errors that may result from bulk normalization methods.                   |
+| `scater`               | [`normalize()`](https://www.rdocumentation.org/packages/scater/versions/1.0.4/topics/normalize)                       | Normalize      | Returns the SingleCellExperiment object with normalized expression values for each cell, using the appropriate size factor and log transformation  |
+| `SingleCellExperiment`               | [`logcounts()`](https://bioconductor.org/packages/devel/bioc/vignettes/scater/inst/doc/vignette-intro.html)           | Logcounts      | Stores or extracts log transformed single cell experiment count data as an assay of the SingleCellExperiment object       |
+| `SingleCellExperiment` | [`SingleCellExperiment()`](https://bioconductor.org/packages/devel/bioc/vignettes/SingleCellExperiment/inst/doc/intro.html#2_creating_singlecellexperiment_instances)                | Single Cell Experiment                     | Creates a `SingleCellExperiment` object                         |
```

Comment:
Good catch. For the other cheatsheets, I made the tables in an order that corresponds with the order the functions appear in the training modules. So it would be a good idea to switch the order her around.